src/pages/heroSection/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Box, Button, Typography } from "@material-ui/core";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+import { heroStyles, heroMdStyles, heroSmStyles } from "./styles";
+import { ImagesCarousel } from "./ImagesCarousel";
+
+const imagesList = [
+  {
+    background: "/img/cards-icons/EOS-Previews-2.webp",
```

Comment:
can we change the order of hero images to leave the one with NFT text for last?

I like the NFT image but I feel the NFT text interferes with the header text a bit

<img width="922" alt="image" src="https://user-images.githubusercontent.com/5632966/180338513-02a81d74-de4d-4e23-92b8-41e35dcd6896.png">

@Edwinjrc  any thoughts?

Reply:
Image order changed

Reply:
Yeah I think that image has too much information. We just wanted to try it, but I think I will talk about it with Johanna and see if we can change it for a more appropiate one.